proto-lens-discrimination/src/Data/ProtoLens/Group.hs

```diff
@@ -0,0 +1,108 @@
+-- | This module implements discriminators of protobufs generically.
+
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE Rank2Types #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+module Data.ProtoLens.Group
+    -- * Comparing Protobuf Messages
```

Comment:
I think this header won't appear in the haddocks unless it's within the (..) of the export list.